build-docker-image/action.yml

```diff
@@ -181,3 +183,15 @@ runs:
     - name: Push ${{ env.DOCKER_REPOSITORY }}:${{ env.IMAGE_TAG }} image to registry
       shell: bash
       run: docker image push --all-tags ${{ env.DOCKER_REPOSITORY }}
+
+    - name: Notify slack channel on job failure
+      uses: rtCamp/action-slack-notify@master
+      if: ${{ failure() && github.ref == 'refs/heads/main' }}
```

Comment:
inputs.slack-webhook is not required. Add a condition not to run when it's not provided?